sandbox/grist/test_dropdown_condition_renames.py

```diff
@@ -111,6 +111,12 @@ def setUp(self):
     ])
     self.assert_invalid_formula_untouched()
 
+  def apply_user_action_ignore_syntax_errors(self, user_action_repr):
```

Comment:
Is this actually still needed and intentional? Doesn't seem right to swallow errors silently.

Reply:
No longer needed. It used to pair with another solution I didn't settle on.